setup.py

```diff
@@ -21,10 +21,14 @@
     author_email='alexey.diyan@gmail.com',
     url='http://github.com/diyan/pywinrm/',
     license='MIT license',
-    packages=('winrm', 'winrm.tests'),
+    packages=('winrm', 'winrm.tests', 'winrm.vendor.requests_kerberos'),
     package_data={'winrm.tests': ['*.ps1']},
     install_requires=['xmltodict', 'requests>=2.9.1', 'requests_ntlm>=0.3.0', 'six'],
-    extras_require = dict(kerberos=['requests-kerberos>=0.10.0'], credssp=['requests-credssp>=1.0.0']),
+    extras_require={
+        'credssp': ['requests-credssp>=1.0.0'],
+        'kerberos:sys_platform=="win32"': ['winkerberos>=0.5.0'],
```

Comment:
From https://github.com/requests/requests-kerberos/blob/master/setup.py#L52 , we may need to include `cryptography>=1.3` as well in this extra section.

Reply:
It's coming in implicitly from requests-ntlm already, but we can add explicitly I guess...

Reply:
Ohh if it is already defined, that works.